test/Database/PostgreSQL/Simple/MigrationTest.hs

```diff
@@ -15,16 +15,42 @@
 
 module Database.PostgreSQL.Simple.MigrationTest where
 
-import           Database.PostgreSQL.Simple           (Connection)
+import           Database.PostgreSQL.Simple           (Connection, execute_)
 import           Database.PostgreSQL.Simple.Migration (MigrationCommand (..),
                                                        MigrationContext (..),
                                                        MigrationResult (..),
                                                        SchemaMigration (..),
                                                        getMigrations,
                                                        runMigration)
-import           Database.PostgreSQL.Simple.Util      (existsTable)
+import           Database.PostgreSQL.Simple.Util      (existsTable, getDataType)
 import           Test.Hspec                           (Spec, describe, it,
                                                        shouldBe)
+metaMigrationSpec:: Connection -> Spec
+metaMigrationSpec con = describe "Meta Migrations" $ do
```

Comment:
The meta migration tests should also ensure that old table rows are equivalent to new table rows. This might be really, really hard to get right. The old values didn't list their timezone, so you need to add one. But what should you use? You can't just use the system's current timezone because of daylight saving, right? Or what if the server has *changed* its settings since the entries were written?

Reply:
True indeed, didn't think of daylight saving... The default time zone is UTC (see https://www.postgresql.org/docs/9.2/runtime-config-client.html#GUC-TIMEZONE), but this could be an issue for some people indeed.

However, if people currently want the true timepoint when the migration was run, they already need to know the time zones for all of them.  With my changes in `polymorphic-return`, people can use the timestamp as `LocalTime` if they want, adding the timezone as before, and nothing breaks.  The only way for breakage to happen is if they change the way the timestamp is used.